services/hack/uploadSubmission.js

```diff
@@ -0,0 +1,43 @@
+const AWS = require("aws-sdk");
+AWS.config.update({
+  region: "ap-southeast-2",
+  accessKeyId: process.env.AWS_ACC_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET
+});
+
+export const upload = async (event, context) => {
+  const s3 = new AWS.S3({ signatureVersion: "v4" });
+  const { fileName } = JSON.parse(event.body);
```

Comment:
I'm not sure I would use a user defined filename, whats stopping me from overriding other hacks submission?

you could/should generate a name ([uuid](https://www.npmjs.com/package/uuid) would do)... this would be easier with my comment on uploading the file through a lambda. so you dont need to return the uuid and then send it back to the other lambda.

Reply:
files are now stored in their own directory within the bucket